pkgs/development/python-modules/django-extensions/default.nix

```diff
@@ -0,0 +1,25 @@
+{
+  stdenv, buildPythonPackage, fetchFromGitHub,
+  vobject, mock, tox, pytestcov, pytest-django, pytest, shortuuid, django
+}:
+
+buildPythonPackage rec {
+  pname = "django-extensions";
+  version = "1.8.1";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "${pname}";
+    repo = "${pname}";
+    rev = "${version}";
+    sha256 = "08rd9zswvjb9dixzyd3p3l3hw3wwhqkgyjvid65niybzjl1xdb5h";
+  };
+
+  buildInputs = [ django vobject mock tox pytestcov pytest-django pytest shortuuid];
```

Comment:
According to https://github.com/django-extensions/django-extensions/blob/b66dad0b4a3525a0f95977d198df87230907a154/setup.py#L130W `six` should probably be places in the `propagatedBuildInputs`.